cmake/Scripts/format.cmake

```diff
@@ -0,0 +1,29 @@
+# Check executable exists
```

Comment:
Is `Scripts` a common CMake term/folder? To me it seems a bit out of place, I'd probably just put it in the `cmake/` folder directly

Reply:
Agreed. I hadn't thought twice about that placement. I'd rather move it to cmake/Format.cmake.